docs/querying/basics.md

```diff
@@ -165,12 +194,12 @@ following units:
 * `s` - seconds
 * `m` - minutes
 * `h` - hours
-* `d` - days - assuming a day has always 24h
```

Comment:
This is needed. It means that we do not take DST, Leap hours, into consideration.

Reply:
I agree, but the adverb "always" should be in front of the verb it is modifying, in this case: 'has'.

assuming a day always has 24h